docs/logical-data-model.md

```diff
@@ -0,0 +1,80 @@
+# Logical data model
+
+![Resource](./assets/logical-data-model-1.png)
+
+## Resource
+
+| Key | Value description |
+| --- | ----------------- |
+| `id` | A unique resource id. |
+| `parentId` | The parent resource id. |
+| `version` | Resource version. |
```

Comment:
In the API we have a 
ResourceSpec {
   name: the porter package name... could also be a generic resource-spec name
   version: resource spec version>
}

Do we want to add this as well?

Reply:
Added